great-idea-website/index.html

```diff
@@ -26,60 +26,87 @@
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </p>
+    </div>
+    <div class='about'>
+      <h3>About</h3>
+      <p>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </p>
+    </div>
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="services">
+      <h3>Services</h3>
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
-  eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
-  eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
-  eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+    <div class="product">
+      <h3>Product</h3>
 
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
-  eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
-  eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+    <div class="vision">
+      <h3>Vision</h3>
 
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
 
-  Copyright Great Idea! 2018
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+  </section>
+  <section class="contact-info">
+    <h3>Contact</h3>
+    <br>
+    <address>
+      123 Way 456 Street
+      Somewhere, USA
+    </address>
+    <br>
+    <phone>
```

Comment:
<phone>  and <email> tags are not a valid HTML tag, instead of you can use span or p tag